src/aiori-IME.c

```diff
@@ -228,9 +257,64 @@ static void IME_Delete(char *testFileName, IOR_param_t *param)
 /*
  * Determine API version.
  */
-static void IME_SetVersion(IOR_param_t *test)
+static char *IME_GetVersion()
+{
+        static char ver[1024] = {};
+#if (IME_NATIVE_API_VERSION >= 120)
```

Comment:
What will happen if `IME_NATIVE_API_VERSION` is not defined? Will it still compile?

Reply:
yes, this is equivalent to: 
`#if defined IME_NATIVE_API_VERSION && if IME_NATIVE_API_VERSION >= 120`
(see https://gcc.gnu.org/onlinedocs/gcc-3.0.2/cpp_4.html#SEC39)

Reply:
Ah, thanks :-)